jdk-annotations/test-annotated/build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id "java-library"
+}
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(21)
+    }
+}
+
+dependencies {
+    compileOnly deps.apt.autoService
+    annotationProcessor deps.apt.autoService
+    compileOnly project(":nullaway")
+    api deps.build.jspecify
+    implementation project(":jdk-annotations:astubx-generator-cli")
```

Comment:
I think rather than this dependence, the `generateAstubx` task should depend on the shadow jar output of `":jdk-annotations:astubx-generator-cli"`?